### PR TITLE
feat: translate to Chinese

### PR DESCRIPTION
**Pull request by CodyTseng**
- translate content into Chinese
- add nostr-relay-nestjs to relay implementation list

### CODE REVIEW

**Comment by erskingardner** [comment]
🤙 I love where this is going!

**Comment by erskingardner** [comment]
@CodyTseng is this ready for review or are there still changes to be made?

**Comment by CodyTseng** [comment]
> @CodyTseng is this ready for review or are there still changes to be made?

Yes, it's ready for review.

File: src/routes/pages/zh/manage-your-sats.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Manage your sats safely
```
Comment (erskingardner) [inline src/routes/pages/zh/manage-your-sats.md:2]: @CodyTseng looks like this one got missed.

**Review by erskingardner — CHANGES_REQUESTED** [review]

File: src/routes/pages/zh/manage-your-sats.md (hunk 1 above)
Comment (erskingardner) [reply src/routes/pages/zh/manage-your-sats.md:2]: actually this page isn't show on the site so you can leave it for now. There is however, one new page "setup-zapping-wallet.md" that needs to be translated when you have time.

Comment (CodyTseng) [reply src/routes/pages/zh/manage-your-sats.md:2]: Okay, I will translate it as soon as possible.

Comment (erskingardner) [reply src/routes/pages/zh/manage-your-sats.md:2]: @CodyTseng did you get a chance to translate this last page? If you don't have time for now, I'll probably merge this PR and you can add that last page in another PR when you have a moment.

Comment (CodyTseng) [reply src/routes/pages/zh/manage-your-sats.md:2]: I have been working a lot recently, and I had a fever 🤒 yesterday. The translation may not be completed recently. I'll finish it on another PR when I have time.

Comment (erskingardner) [reply src/routes/pages/zh/manage-your-sats.md:2]: No worries at all! Take your time and I hope you feel better soon.

**Pull request merged**